applications/luci-app-frpc/htdocs/luci-static/resources/view/frpc.js

```diff
@@ -212,6 +212,16 @@ return view.extend({
 		s.option(form.Value, 'type', _('Proxy type')).modalonly = false;
 		s.option(form.Value, 'local_ip', _('Local IP')).modalonly = false;
 		s.option(form.Value, 'local_port', _('Local port')).modalonly = false;
+		o = s.option(form.Value, 'remote_port', _('Remote port'));
+		o.modalonly = false;
+		o.depends('type', 'tcp');
+		o.depends('type', 'udp');
+		o.cfgvalue = (function(cfgvalue, _this){
+			return function() {
+				var v = cfgvalue.apply(_this, arguments);
+				return v&&v!='0'?v:'#';
```

Comment:
? Can't it be simpler? No one understands that

Reply:
Just a javascript closure, for reducing local variables.
This is equals to:
```
var oldcfgvalue = o.cfgvalue;
o.cfgvalue = function() {
  var v = oldcfgvalue.apply(o, arguments);
  return v&&v!='0'?v:'#';
};
```

Reply:
Is there any way to wrap `cfgvalue` function?

Reply:
I'm sorry, but every time I look at this line, I have to think what this is doing.
Since I read a lot of code with different language, I prefer it to make this expression easier. 
Regardless of whether we reduce local variables, I think the readability is more important from my point of view!

Reply:
I found that `this.super` fit my situation, it's simple now